components/locid/src/helpers.rs

```diff
@@ -136,7 +142,7 @@ impl<T> From<Vec<T>> for ShortVec<T> {
             0 => ShortVec::ZeroOne(None),
             #[allow(clippy::unwrap_used)] // we know that the vec is not empty
             1 => ShortVec::ZeroOne(Some(v.into_iter().next().unwrap())),
-            _ => ShortVec::Multi(v),
+            _ => ShortVec::Multi(v.into_boxed_slice()),
```

Comment:
Maybe update this to `impl<T> From<Box<[T]>>`? If we have a `Box` we don't need to go through `Vec`.

Reply:
Not changing to `Box<[T]>` as discussed offline.